style.css

```diff
@@ -0,0 +1,188 @@
+body { font-family: "3rd party font", serif; }
+
+.Mobile_background {
+  width: 365px;
+  height: 5410px;
```

Comment:
- [ ] [Optional] The whole purpose of this exercise is to show you how to design a responsive webpage. Using fixed values in width and height properties will make it really really hard to accomplish this kind of design. I strongly suggest you avoid using any `width` and `height` property with a pixel value.